lnwallet/fee_estimator.go

```diff
@@ -474,3 +477,50 @@ type WebAPIFeeSource interface {
 	// specifics are left to the WebAPIFeeSource implementation.
 	ParseResponse(r io.Reader) (map[uint32]uint32, error)
 }
+
+// ExternalFeeSource is an implementation of the WebAPIFeeSource that utilizes
+// a user-specified fee estimation API for Bitcoin. It expects the response
+// to be in the JSON format: `fee_by_block_target: { ... }` where the value maps
+// block targets to fee estimates (in sat per kilovbyte).
+type ExternalFeeSource struct {
```

Comment:
Naming suggestion: `SparseConfFeeSource`? Sparse in that it only includes non-redundant confirmation targets.

Reply:
That it's external is already a given since the interface itself is called `WebAPIFeeSource`.

Reply:
fixed